Files/Dialogs/SettingsDialog.xaml.cs

```diff
@@ -29,13 +29,22 @@ private void Current_SizeChanged(object sender, WindowSizeChangedEventArgs e)
 
         private void UpdateDialogLayout()
         {
-            if (Window.Current.Bounds.Height <= 600)
+            if (Window.Current.Bounds.Height <= 640)
```

Comment:
I would change this part of the code (lines 32-39) with this:


> if (Window.Current.Bounds.Height <= 710)
            {
                ContainerGrid.Height = Window.Current.Bounds.Height - 70;
            }
            else
            {
                ContainerGrid.Height = 640;
            }


The reason is that when the window height is low it will get overlapped the x of the program itself with the settings Window, and the last field will got cut:

Before:
![image](https://user-images.githubusercontent.com/11770760/141681321-61753d28-81e2-4c08-8420-e1b4df5b9c1a.png)
![before](https://user-images.githubusercontent.com/11770760/141681366-62d847d9-389b-4e37-aaa4-50dc6f63233a.gif)

After:
![image](https://user-images.githubusercontent.com/11770760/141681134-3111cd52-d257-4160-9993-dc73886d5730.png)
![After](https://user-images.githubusercontent.com/11770760/141681191-2e7de000-0912-4b38-a31d-f4990a0e8dd4.gif)